index.scss

```diff
@@ -20,7 +20,7 @@
   position: absolute;
   padding-top: 11px;
   top: 0;
-  left: -4px;
+  left: -3px;
```

Comment:
The 4px was to offset the 4px border. I was putting the alert inside a React-Modal, and there the icon's 1px background was coming out to the left. I made modifications to the container to fix it.